apps/judicial-system/web-e2e/src/integration/Defender/caseOverview.ts

```diff
@@ -0,0 +1,34 @@
+import { DEFENDER_ROUTE } from '@island.is/judicial-system/consts'
+import {
+  Case,
+  CaseOrigin,
+  CaseState,
+  CaseType,
+} from '@island.is/judicial-system/types'
+
+import { intercept } from '../../utils'
+
+describe('Defender case overview', () => {
+  beforeEach(() => {
+    cy.stubAPIResponses()
+  })
+
+  it('should have a title', () => {
+    const theCase: Case = {
+      id: 'test_id',
+      created: '2020-09-16T19:50:08.033Z',
+      modified: '2020-09-16T19:51:39.466Z',
+      state: CaseState.ACCEPTED,
+      origin: CaseOrigin.RVG,
+      type: CaseType.CUSTODY,
+      policeCaseNumber: '007-2021-202000',
+    }
+
+    cy.stubAPIResponses()
```

Comment:
You are calling stubAPIResponses twice. It's only needed in the beforeEach

Reply:
You are right, will fix. I noticed this is done in a lot of e2e tests though.